examples/uikit-app-shell/vite.config.ts

```diff
@@ -17,6 +17,9 @@ export default defineConfig(({ mode }) => ({
       },
     }),
     tsconfigPaths(),
-    HvAppShellVitePlugin({ mode }),
+    HvAppShellVitePlugin({
+      mode,
+      modules: ["src/pages/Home", "src/pages/Page2"],
```

Comment:
Since this is a simple/minimal example, we could leverage the `autoMenu` and `autoViewsAndRoutes` features from App Shell - simplifying this and the config file

```suggestion
      autoMenu: true,
      autoViewsAndRoutes: true,
```

Reply:
Yup, I'll use these options 🙏